sdk/sdk/Models/dao/OBAModelDAOUserPreferencesImpl.m

```diff
@@ -243,8 +235,6 @@ - (void) writeSetRegionAutomatically:(BOOL)setRegionAutomatically {
     NSUserDefaults * user = [NSUserDefaults standardUserDefaults];
     NSNumber * v = @(setRegionAutomatically);
     [user setObject:v forKey:kSetRegionAutomaticallyKey];
-
-    [[GAI sharedInstance].defaultTracker set:[GAIFields customDimensionForIndex:2] value:(setRegionAutomatically ? @"YES" : @"NO")];
```

Comment:
@bbodenmiller It is a valid concern.  There are better ways of doing this via hiding the concrete model objects via proper abstractions and extracting it from App Delegate, but those would require more significant changes.

I am open to restoring the logic and postponing the GA dependency removal to a different PR